docs/UserGuide.md

```diff
@@ -8,22 +8,23 @@ the desire to **store**, **access** and **share** your recipes easily?
 
 ***RIZZ***ipe is a **command-based recipe database** that is designed just for chefs _LIKE YOU_ to manage recipes at the touch of a finger!
 
-RIZZipe stars 3 main features:
+***RIZZ***ipe comes equipped with multiple features but stars **3** main ones:
```

Comment:
with multiple features, but we love these 3 the most:

Reply:
good suggestion